src/sage/schemes/elliptic_curves/ell_curve_isogeny.py

```diff
@@ -3345,6 +3348,72 @@ def _composition_impl(left, right):
         return NotImplemented
 
 
+def compute_isogeny_bmss(E1, E2, l):
+    r"""
+    Compute the kernel polynomial of the unique normalized isogeny
+    of degree ``l`` between ``E1`` and ``E2``.
+
+    ALGORITHM: [BMSS2006], algorithm *fastElkies'*.
+
+    AUTHORS: Rémy Oudompheng.
+    `Original code <https://github.com/remyoudompheng/isogeny_weber/blob/64289127a337ac1bf258b711e02fea02b7df5275/isogeny_weber/isogenies.py#L272-L332>`
+    released under the
+    `MIT license <https://github.com/remyoudompheng/isogeny_weber/blob/64289127a337ac1bf258b711e02fea02b7df5275/LICENSE>`.
+    Slightly adjusted for inclusion in the Sage library.
+
```

Comment:
It is not that the implementor of an algorithm does not deserve a credit, but I think giving the credit (and honoring) him/her in the docstring should be refrained. I think so because it does not give the user any useful information. All algorithms implemented in Sage has an implementor, but we do not attach his/her name to the docstring. Perhaps that is why the sage developer guide does not mention such a practice.

So I suggest to move the implementor information to the code, as a comment following the docstring.

On the other hand, if `compute_isogeny_bmss` is internal, that is, it is `_compute_isogeny_bmss`, then I would not care.

Reply:
Alright, done.

(Some other functions do have an `AUTHORS:` or even `AUTHOR:` block, but indeed the developer guide only supports having one in the docstring for the entire file.)